src/test/java/io/github/eocqrs/kafka/consumer/KfConsumerTest.java

```diff
@@ -42,16 +48,16 @@
  *
  * @since 0.0.0
  */
+@ExtendWith(MockitoExtension.class)
 final class KfConsumerTest {
 
   @Test
-  void subscribes() throws Exception {
-    final Consumer<String, String> consumer =
-      new KfConsumer<>(
-        new KfXmlFlexible<String, String>(
-          "consumer.xml"
-        ).consumer()
-      );
+  void subscribes(
+    @Mock final ConsumerSettings<String, String> settingsMock,
```

Comment:
I don't tolerate compound var naming.
settings and consumer sounds a little bit better

`@Mock` already indicating that they are mocks